driver/apdu/mbim.c

```diff
@@ -107,6 +107,10 @@ static int select_sim_slot(struct mbim_data *mbim_priv)
         if (is_sim_available(mbim_priv)) {
             return 0;
         }
+        struct timespec ts;
+        ts.tv_sec = 0;
+        ts.tv_nsec = 100000;
```

Comment:
```suggestion
        struct timespec ts = {
            .tv_sec = 0,
            .tv_nsec = 100000
        };
```
It can even be rewritten shortly as
```c
nanosleep(&(struct timespec){ .tv_sec = 0, .tv_nsec = 100000 }, NULL);
```
But the readability is not good.
And unfortunately, the C standard replacement `thrd_sleep(3)` is only available since C11.

Reply:
Thanks, I more or less assumed we couldn't do this.

Reply:
Nah, I was wrong. The communication introduces significant delay, it just wasn't significant enough. I'll change the length of the wait to the intended value.